### ISSUE TITLE
Ensure Support for Items with Translation Field Data

### ISSUE THREAD

**Issue by sourslaw** [UMNLibraries/cdm-blacklightify#137]
## Descriptive Summary
- A number of items have a translation field which is a translation of the transcription field. For these items, the transcription field is actually in a foreign language. The translation is an english translation of the transcription field data.

  - <img width="1003" height="375" alt="Image" src="https://github.com/user-attachments/assets/0f6f8f37-2edb-45eb-99cd-924bbfcb023d" />

  - <img width="1004" height="356" alt="Image" src="https://github.com/user-attachments/assets/60d2adb2-8dc8-4a87-9d59-9e661bcda0e3" />

- [example json](https://umedia.lib.umn.edu/item/p16022coll264:133.json)

## Expected Behavior
- Items which have translated transcript data have translation display functionality

## Notes
- Example item: [Alessandro Sisca (Riccardo Cordiferro) - letter to Lucia Fazio, 1900-10-18
](https://umedia.lib.umn.edu/item/p16022coll264:133)

**Comment by sourslaw** [comment]
# Items for confirmation

Sonia Cancian interviewing Antonietta Petris, 2011-09-20
- p16022coll264:0
- A/V item with transcription field only, no translation field
- This permutation has never been accounted for. Code previously assumed all items with transcripts were text  and had children

Diego Delfino - Letter to his Daughter, 1925-03-26
- p16022coll264:3

Letter to Philip Hitti from his brother (Joseph) Yusuf Hitti, 1941
- p16022coll264:1
- Text items with transcription and translation fields in single parent only, no children

Alessandro Sisca (Riccardo Cordiferro) - letter to Lucia Fazio, 1900-10-18
- p16022coll264:133
- Compound text item with transcription and translation fields in child docs

Interview with Hyman Berman
- p16022coll171:3699
- A/V item with transcription field and pdf attachment of transcript
- This previously only displayed the pdf tab, but now displays both that and a transcript tab
